content/docs/start/data-management/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Get Started: Data Management'
+description:
+  'Get started with DVC data management. Version your data, build data pipelines
+  and learn about pipeline metrics, parameters and plots.'
+---
+
+# Get Started: Data Management
+
+## Chapters
+
+- **[Data and model versioning]** - Manage large files, datasets, and machine
+  learning models. Track your data and couple its versions to your code
+  versions, while keeping it stored properly outside of your Git repo.
+
+- **[Data pipelines]** - Use pipelines to describe how models and other data
+  artifacts are built, and provide an efficient way to reproduce them. Think
+  "Makefiles for data and ML projects" done right.
+
+- **[Metrics, parameters, and plots]** - Those are first class citizens in DVC
+  pipelines. Capture, evaluate, and visualize ML projects without leaving Git.
+
+[data and model versioning]: /doc/start/data-management/data-versioning
+[data pipelines]: /doc/start/data-management/data-pipelines
+[metrics, parameters, and plots]:
+  /doc/start/data-management/metrics-parameters-plots
+
+<admon type="tip">
+
+The steps and results of some of these chapters are captured in our
+[example-get-started] repo. Feel free to `git clone/checkout` any of its
+[tags][example-get-started-tags].
+
+[example-get-started]: https://github.com/iterative/example-get-started
+[example-get-started-tags]:
+  https://github.com/iterative/example-get-started/tags
+
+</admon>
+
+## Where To Go Next
+
+Pick a page from the list above, the left-side navigation bar, or just click
+`NEXT` below!
```

Comment:
Do you think it's worth referring back to https://dvc.org/doc/start#get-started-with-dvc somewhere in these pages? No strong opinion, so leaving it up to you.

Reply:
No strong opinion as well, why not :) ! I'll add 👍